Customer Support Procedures/Neo4j-v5.0-migration.md

```diff
@@ -44,4 +44,8 @@ recreate them as RANGE indexes after the migration, without any inconvenience.
 - migrate your own BTREE indexes and then use the `--force-btree-indexes-to-range` option of the `neo4j-admin database migrate`
 command to migrate the Linkurious Enterprise indexes.
 
+When Linkurious reconnect to the data-source after the migration has completed, it is going to detect that the database
```

Comment:
@gmasclet is this because the db id is changed like in the migration between neo4j v3 to v4 or because you have to reimport all the data? In the second case, just dismissing the warning is not safe since all the node & edges references are broken.

Reply:
@fantandrea it's just because the database identifier is changed as a result of the migration. The node & edges references aren't broken, see https://linkurious.atlassian.net/browse/LKE-7689?focusedCommentId=33836

Reply:
Thanks for the extra test and clarification!